administrator/components/com_fields/src/Model/FieldModel.php

```diff
@@ -1108,6 +1109,12 @@ function () use ($component, $section) {
             }
         }
 
+        $componentInterface = Factory::getApplication()->bootComponent($component);
+
+        if ($componentInterface instanceof FieldsServiceInterface) {
+            $componentInterface->prepareForm($form, $data);
+        }
```

Comment:
```suggestion
        $componentBooted = Factory::getApplication()->bootComponent($component);

        if ($componentBooted instanceof FieldsServiceInterface) {
            $componentBooted->prepareForm($form, $data);
        }
```

You don't have an interface in the variable so calling it ...Interface is missleading.